TipBot/Settings.cs

```diff
@@ -6,23 +6,48 @@ public class Settings
     {
         public void Initialize(TextFileConfiguration configReader)
         {
+            this.BotToken = configReader.GetOrDefault<string>("token", "discordtoken");
+
+			// To run stratis daemon that supports RPC use "dotnet exec ...\netcoreapp2.1\Stratis.StratisD.dll -rpcuser=user -rpcpassword=4815162342 -rpcport=23521 -server=1"
+			this.DaemonUrl = configReader.GetOrDefault<string>("daemonUrl", "http://127.0.0.1:23521/");
```

Comment:
This was removed after creating interface to abstract implementation for the node connections, you've reintroduced it.